compiler/pytest.vim

```diff
@@ -2,10 +2,19 @@
 " Compiler: Py.test <http://pytest.org/>
 " Maintainer: Whyme Lyu <5longluna at gmail.com>
 " URL: https://github.com/5long/pytest-vim-compiler
+" Last change by: Arthur Jaron <hifreeo@gmail.com>
+" Last change: 31.05.2014 
 
 if exists("current_compiler")
   finish
 endif
+
+if has("python3") && has("python")
+  CompilerSet makeprg=py.test-3.4\ --tb=short\ -q
+elseif !has("python3") && has("python")
+  CompilerSet makeprg=py.test-2.7\ --tb=short\ -q
+endif
+
```

Comment:
If I recall correctly, the commonly accepted best practice of python development is just use virtualenv and don't bother with this. Beside, the `has("python3")` function of vim means "you can write vim plugins in python3", not "you are running in a python environment with py.test installed".

So um, nope. I don't think it's worth adding extra code for this Python 2/3 detection.